### DISCUSSION TITLE
Can't create codespace from Laravel Sail config

### DISCUSSION THREAD

**Discussion by garrettw** [community/community#48069] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I recently created a repo using Laravel Sail, and I found that Codespaces is not able to create a setup from that config.
I created an issue on Laravel Sail about it ( https://github.com/laravel/sail/issues/550 ) but I'm not sure if the problem lies with how the config is written, or if it's a bug in Codespaces. All related info is posted in the issue I linked.

**Comment by ijpatricio** [comment, marked as answer]
Hey @garrettw , I just left you some directions on your issue: https://github.com/laravel/sail/issues/550#issuecomment-1553767784

I hope this helps others as well 👍
